.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+
+        for issue in self.merged_issues:
+            prioritized = self._calculate_priority(issue)
+            self.prioritized_issues.append(prioritized)
+
+        # 우선순위별 정렬 (Critical > Important > Suggestion)
+        priority_order = {"Critical": 0, "Important": 1, "Suggestion": 2}
+        self.prioritized_issues.sort(
+            key=lambda x: (priority_order.get(x.priority, 3), -x.vote_count)
+        )
+
+        print(f"✅ 우선순위 계산 완료")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)